src/theme.ts

```diff
@@ -224,12 +224,19 @@ const defaultTheme: ThemeInterface = {
     gutter: '2px',
   },
   rightPanel: {
-    backgroundColor: '#263238',
+    backgroundColor: '#3d434a', //'#263238',
     width: '40%',
-    textColor: '#ffffff',
+    textColor: '#ffffff'
   },
   codeBlock: {
     backgroundColor: ({ rightPanel }) => darken(0.1, rightPanel.backgroundColor),
+    colors: {
+      symbol: ({rightPanel}) => rightPanel.textColor,
+      property: ({codeBlock}) => codeBlock.colors.symbol,
```

Comment:
Default `white`.

Reply:
To keep backward compatibility it has to inherit from rightPanel.textColor 😅.